### PR TITLE
detect history API support as modernizer does

### PR DESCRIPTION
**Pull request by pazams**
this closes https://github.com/devote/HTML5-History-API/issues/82

### CODE REVIEW

File: history.js [hunk 1]
```diff
@@ -472,6 +472,29 @@
...
+        if ((ua.indexOf('Android 2.') !== -1 ||
+            (ua.indexOf('Android 4.0') !== -1)) &&
+            ua.indexOf('Mobile Safari') !== -1 &&
+            ua.indexOf('Chrome') === -1 &&
+            ua.indexOf('Windows Phone') === -1) {
+          return false;
+            }
+
+        // Return the regular check
+        return (window.history && 'pushState' in window.history);
```
Comment (devote) [inline history.js]: maybe so:
`return !!historyPushState;`

**Comment by pazams** [comment]
@devote, thanks for your comment. PR updated.

**Comment by devote** [comment]
Please add this code to `history.ielte7.js`
Thanks!

**Comment by pazams** [comment]
Done
Thanks

**Pull request merged**